app/containers/Forms/ProductField.tsx

```diff
@@ -27,11 +27,11 @@ export const ProductFieldComponent: React.FunctionComponent<Props> = (
 ) => {
   const {formData, query, onChange} = props;
 
-  const productIsActive = (formData, query) => {
+  const productIsPublished = (formData, query) => {
     const product = query.allProducts.edges.find(
       ({node}) => node.rowId === formData.productRowId
     )?.node;
-    if (product?.state === 'active' || !product) return true;
+    if (product?.productState === 'published' || !product) return true;
     return false;
```

Comment:
You could write this like:
```js
return query.allProducts.edges.some(
  ({node}) => node.rowId === formData.productRowId && node.state === 'active'
);
```

Reply:
doesn't `.some` return true if any item in the array matches the requirements? So this would return true if any node matched an id and had an 'active' state? ie:
[
{rowId: 1, state: 'active'}, rowId = productRowId = 1 && state = 'active' => true 
{rowId: 2, state: 'archived'}, rowId = productRowId = 2 && state = 'archived' => false
{rowId: 2, state: 'archived'} => rowId = productRowId = 3 && state = 'archived' => false
]
.some on the above array would return: [true, false, false] === true?

Reply:
`productRowId` is a parameter, so only one node would match the product id

Reply:
changing it still appears to have a different effect than how it's currently written. can we leave it as is?

Reply:
sure, just put a TODO comment